pkg/storage/storagewrappers/context.go

```diff
@@ -9,57 +9,67 @@ import (
 	"github.com/openfga/openfga/pkg/storage"
 )
 
-// ContextTracerWrapper is a wrapper around a datastore that passes a new
-// context to the underlying datastore methods.
-// This is so that if the original context gets cancelled (e.g by the client), the underlying database connection isn't closed.
-// So, we let outstanding queries run their course even if the original context gets cancelled to avoid database connection churning.
+// ContextTracerWrapper is a wrapper for a datastore that introduces a new context to the underlying datastore methods.
+// Its purpose is to prevent the closure of the underlying database connection in case the original context is cancelled,
+// such as when a client cancels the context. This ensures that ongoing queries are allowed to complete even if the
+// original context is cancelled, helping to avoid unnecessary database connection churn.
 //
-// ContextTracerWrapper must be the first wrapper around the datastore if traces are to work properly.
+// It is crucial for ContextTracerWrapper to be the first wrapper around the datastore for traces to function correctly.
```

Comment:
Can we move this godoc to above `NewContextWrapper` which is when it is most likely to be read?